### ISSUE TITLE
Ops Ground Control Post-Merge (8-Jun-20) Issues List

### ISSUE THREAD

**Issue by disgruntled-patzer** [yonahbox/Yonah_ROS_packages#15]
This issue serves as a catch-all for all issues that arise during the two weeks of system-level bench testing, following the first round of merges on 8 Jun 2020.

**Issues:**

Air side:

- [x] Syntax errors in Air Despatchers 
- [x] Removal of debug information that SBD node passes to air despatcher
- [x] Handing control of sleep interval over from SBD node to air despatcher
- [x] Unicode Decode Error thrown when accessing rockBlock.py on Beaglebone SBC

Ground side:

- [x] No message header interpretation for statustext
- [x] Typo error when sending a message to the telegram link
- [x] Interpreting of debug information that SBD node passes to gnd despatcher
- [x] Catch the TypeError thrown by Ground SBD Node when web server is not running
- [x] Swap checks for time and whitelisted IMEI around for Ground SBD Node to prevent spam when an unknown msg is placed in the server
- [x] RQT signal-slot errors upon reception of incoming non standard msgs
- [x] RQT has no function to subscribe to `ogc/yonahtext` (which handles statustext msgs)
- [x] Pass message headers from statustext to RQT (currently, the YonahText msg does not include the headers)

Misc:

- Setup issues: Will be reflected on the Yonah_scripts [Bonesetup folder](https://github.com/yonahbox/Yonah_scripts/tree/systemd/Bonesetup).

**Comment by disgruntled-patzer** [comment]
First round of fixes applied here: https://github.com/yonahbox/Yonah_ROS_packages/pull/16

**Comment by disgruntled-patzer** [comment]
Raw breakdown of SBD Node Error in Bone:

```
Traceback (most recent call last):
  File "/home/ubuntu/Yonah_ROS_packages/ogc_ws/src/sbd/src/sbd_air_link.py", line 27, in <module>
    import rockBlock
  File "/home/ubuntu/Yonah_ROS_packages/ogc_ws/devel/lib/python2.7/dist-packages/rockBlock/__init__.py", line 35, in <module>
    exec(__fh.read())
  File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 13850: ordinal not in range(128)
[sbd_air_link-3] process has died [pid 1962, exit code 1, cmd /home/ubuntu/Yonah_ROS_packages/ogc_ws/src/sbd/src/sbd_air_link.py __name:=sbd_air_link __log:=/home/ubuntu/.ros/log/fedde1ce-aef0-11ea-91dd-0c1c57f57c36/sbd_air_link-3.log].
log file: /home/ubuntu/.ros/log/fedde1ce-aef0-11ea-91dd-0c1c57f57c36/sbd_air_link-3*.log
```

**Comment by disgruntled-patzer** [comment]
RQT issues solved in #20

**Comment by disgruntled-patzer** [comment]
Statustext issues solved in #25

**Comment by disgruntled-patzer** [comment]
SBD issues solved in #28 . With that, first round of testing and fixes are completed. 2nd round of testing will take place alongside our migration to Ubuntu 20.04
